media-server/config.txt

```diff
@@ -0,0 +1,12 @@
+CAMERA='/dev/video0'
+CAMERA_WIDTH=1280
+CAMERA_HEIGHT=720
+CAMERA_FORMAT=YUY2
+MODEL_LOCATION=graph_tinyyolov3_tensorflow.pb
+INPUT_LAYER='inputs'
+OUTPUT_LAYER='output_boxes'
+LABELS='labels.txt'
+OUTPUT_FILE='recording.mp4'
+HOST=localhost
```

Comment:
This is ambiguous, does it refer to the input streaming or the output streaming?